src/tasks/WasmAppBuilder/WasmAppBuilder.cs

```diff
@@ -251,65 +221,43 @@ public override bool Execute ()
 
         config.Assets.Add(new VfsEntry ("dotnet.timezones.blat") { VirtualPath = "/usr/share/zoneinfo/"});
 
-        if (RemoteSources?.Length > 0) {
+        if (RemoteSources?.Length > 0)
+        {
             foreach (var source in RemoteSources)
                 if (source != null && source.ItemSpec != null)
                     config.RemoteSources.Add(source.ItemSpec);
         }
 
-        using (var sw = File.CreateText(Path.Join(AppDir, "mono-config.js")))
+        string monoConfigPath = Path.Join(AppDir, "mono-config.js");
+        using (var sw = File.CreateText(monoConfigPath))
         {
             var json = JsonSerializer.Serialize (config, new JsonSerializerOptions { WriteIndented = true });
             sw.Write($"config = {json};");
         }
+        _fileWrites.Add(monoConfigPath);
 
-        using (var sw = File.CreateText(Path.Join(AppDir, "run-v8.sh")))
+        string runv8Path = Path.Join(AppDir, "run-v8.sh");
```

Comment:
Does it make sense to write this file outside of the WasmAppBuilder and just pass it along as a property?  Something like `ExtraFiles` or whatever other name makes sense.

Reply:
I'm not sure of the use case of `run-v8.sh`. Would it be used in a context that doesn't use WasmAppBuilder?

Reply:
that's just a helper script to start, in our case, tests against v8.  I think it's an outside of WasmAppBuilder concern.

Reply:
I'll put up this change in a follow up PR